multiversx_sdk/core/transactions_factories/token_management_transactions_factory_test.py

```diff
@@ -165,6 +166,22 @@ def test_create_transaction_for_setting_special_role_on_fungible_token():
     assert transaction.value == 0
 
 
+def test_set_all_roles_on_fungible_token():
+    transaction = factory.create_transaction_for_setting_special_role_on_fungible_token(
+        sender=frank,
+        user=grace,
+        token_identifier="FRANK-11ce3e",
+        add_role_local_mint=True,
+        add_role_local_burn=True,
+        add_role_esdt_transfer_role=True
+    )
+
+    assert transaction.data
+    assert transaction.data.decode() == "setSpecialRole@4652414e4b2d313163653365@1e8a8b6b49de5b7be10aaa158a5a6a4abb4b56cc08f524bb5e6cd5f211ad3e13@45534454526f6c654c6f63616c4d696e74@45534454526f6c654c6f63616c4275726e@455344545472616e73666572526f6c65"
```

Comment:
i think it might be cleaner to have this expected value decomposed, to be able to see more easily that there is also `ESDTTransferRole` encoded

Reply:
maybe we can evaluate to have this pattern in all tests

Reply:
Done